lib/model/store.dart

```diff
@@ -263,6 +270,26 @@ abstract class GlobalStore extends ChangeNotifier {
 
   Account? getAccount(int id) => _accounts[id];
 
+  /// The account the user has visited most recently, if available.
+  ///
+  /// This method retrieves the stored account ID from the database and returns
+  /// the corresponding account on the device in the following way:
+  ///   - If no ID is stored in the database (i.e., no account is visited yet),
+  ///     [OptionNone] is returned.
+  ///   - If the stored ID doesn't correspond to any account on the current
+  ///     device (i.e., the account was logged out and no other account was
+  ///     visited in the same running session), [OptionSome(value: null)]
+  ///     is returned.
+  ///   - Otherwise, [OptionSome] with the corresponding account is returned.
+  Option<Account?> get lastVisitedAccount {
+    final value = settings.getInt(IntGlobalSetting.lastVisitedAccountId);
+    return value == null ? OptionNone() : OptionSome(_accounts[value]);
+  }
+
+  Future<void> setLastVisitedAccount(int? accountId) {
```

Comment:
nit: can make `accountId` required, since we don't plan to support unsetting the value